File: app/src/main/java/balti/migratehelper/ComposeSmsActivity.java
package balti.migratehelper;

import android.os.Bundle;
import android.support.annotation.Nullable;
import android.support.v7.app.AppCompatActivity;
import android.widget.Toast;

public class ComposeSmsActivity extends AppCompatActivity {

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        Toast.makeText(this, R.string.please_change_sms_app, Toast.LENGTH_LONG).show();
        finish();
    }
}

File: app/src/main/java/balti/migratehelper/dummies/ComposeSmsActivity.kt
package balti.migratehelper.dummies

import android.os.Bundle
import android.widget.Toast
import androidx.appcompat.app.AppCompatActivity
import balti.migratehelper.R

class ComposeSmsActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        Toast.makeText(this, R.string.please_change_sms_app, Toast.LENGTH_LONG).show()
        finish()
    }
}